readthedocs_build/config/test_config.py

```diff
@@ -349,6 +349,165 @@ def it_uses_validate_file(tmpdir):
             assert args[0] == 'setup.py'
 
 
+def describe_validate_submodules():
+
+    def it_defaults_to_all_submodules(tmpdir):
+        with tmpdir.as_cwd():
+            source_file = tmpdir.join('subdir', 'readthedocs.yml')
+            build = get_build_config({}, source_file=str(source_file))
+            build.validate_submodules()
+            # TODO: This will be the dafult behavior?
```

Comment:
What I mean here is: on other parts of the code, when the key is not given, it doesn't get added to the build dict, and then the default values are chosen by rtd, I think with that current behavior we are repeating some logic.

Reply:
small typo: *default*